stubs/beautifulsoup4/bs4/element.pyi

```diff
@@ -262,7 +262,8 @@ class Tag(PageElement):
     def __copy__(self: Self) -> Self: ...
     @property
     def is_empty_element(self) -> bool: ...
-    isSelfClosing = is_empty_element
+    @property
+    def isSelfClosing(self) -> bool: ...
```

Comment:
Hmm, this one is actually less accurate to the runtime, maybe mypy / stubtest is missing a trick

Reply:
Mypy interprets aliases of methods inside class bodies as instance variables. I noticed it a while back when working on a stubtest patch,but forgot to file an issue about it.